internal/lockclient/simple_client.go

```diff
@@ -24,18 +30,60 @@ var _ Client = (*SimpleClient)(nil)
 
 // SimpleClient implements Client, the lockclient for Lockey.
 type SimpleClient struct {
+	Id     uint8
+	config *SimpleConfig
 }
 
 // Acquire makes a HTTP call to the lockserver and acquires the lock.
 // The errors invloved may be due the HTTP errors or the lockservice errors.
 func (sc *SimpleClient) Acquire(d lockservice.Descriptors) error {
-	panic("TODO")
+	baseUrl := sc.config.IPAddr + ":" + sc.config.PortAddr
+	endPoint := baseUrl + "/acquire"
+
+	var jsonStr = []byte(`{"FileID":` + string(d) + "}")
+	req, err := http.NewRequest("POST", endPoint, bytes.NewBuffer(jsonStr))
+	req.Header.set("Content-Type", "application/json")
+
+	client := &http.Client{}
+	resp, err := client.Do(req)
+
+	if err != nil {
+		panic(err)
+	}
+	defer resp.Body.Close()
+
+	body, _ := ioutil.ReadAll(resp.Body)
+
+	if string(body) == "lock acquired" {
```

Comment:
Maybe we can have better implementation of a confirmation? Maybe change the loockservice to provide a better output?
Lets discuss this.

Reply:
Same for below.

Reply:
Yeah this I thought was needed while writing this if statement

Reply:
In `checkacquire` and `checkrelease`, we need to change the status of the response back to the client to something other than 200 as well.